portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -34,25 +38,19 @@
 @WebServlet("/list-comments")
 public class ListCommentsServlet extends HttpServlet {
 
+  private static final Gson gson = new Gson();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int maxNumComments = Integer.parseInt(request.getParameter("max"));
```

Comment:
Same as before, you can set the `maxNumComments` to a default value or return an empty list if the parameter does not exist or is not valid.

Reply:
fixed